appinfo/info.xml

```diff
@@ -9,6 +9,6 @@
     <namespace>Survey_Server</namespace>
     <category>other</category>
     <dependencies>
-        <nextcloud min-version="9" max-version="13"/>
+        <nextcloud min-version="9" max-version="14"/>
```

Comment:
Forgot to submit my review.

I'd just lift the min. to the last supported version: 13